src/components/UserFeatures.js

```diff
@@ -120,68 +120,88 @@ function UserFeatures({ scAddress, userAddress, quorem, isOwner }) {
+      <div>
+        <div className="user-section mt-4">
+          <h1>User</h1>
+          <Form className="mt-4">
+            <Form.Group
+              className="mb-3 d-flex user-label"
+              controlId="formUserAddress"
+            >
+              <Form.Label className="user-address">User address:</Form.Label>
+              <Form.Control
+                className="mt-1 responsive"
+                type="text"
+                placeholder={userAddress}
+                plaintext
+                readOnly
+              />
+            </Form.Group>
             <div>
-              <a
-                href={`https://etherscan.io/tx/${writeData?.hash}`}
-                target="_blank"
-                rel="noreferrer"
-              >
-                Etherscan
-              </a>
+              <Form.Group className="mb-3" controlId="formDepositEther">
+                <Form.Label>Deposit Ether</Form.Label>
+                <Form.Control
+                  className="deposit-input-field"
+                  type="number"
+                  step="0.000001"
+                  placeholder="Enter amount in ether"
+                  onChange={onChangeDeposit}
+                />
+              </Form.Group>
+
+              <div className="user-button">
+                <Button
+                  className="deposit-button"
+                  disabled={!depositWrite || depositIsLoading}
+                  variant="danger"
+                  onClick={() => depositWrite?.()}
+                >
+                  {depositIsLoading ? "Depositing..." : "Deposit"}
+                </Button>
+              </div>
             </div>
+
+            {depositIsSuccess && (
+              <div>
+                Successfully deposited {depositAmt} ETH!
+                <div>
+                  <a
+                    href={`https://etherscan.io/tx/${writeData?.hash}`}
+                    target="_blank"
+                    rel="noreferrer"
+                  >
+                    Etherscan
+                  </a>
+                </div>
+              </div>
+            )}
+            {(prepareDepositIsError || depositIsError) && (
+              <div className="custom-word-wrap">
+                Error: {(prepareDepositError || depositError)?.message}
+              </div>
+            )}
+          </Form>
+        </div>
+        <div className="transaction-section">
+          <div className=" mt-6">
+            <h2 className="fs-2">Transaction Approval List</h2>
+            {readIsLoading ? (
+              <p>Loading transaction list...</p>
+            ) : (
+              <ul>
+                {unapprovedTxns.map((txn) => (
+                  <li key={txn.id}>{`id ${txn.id} || To=${
+                    txn?.to
+                  } || Amount: ${formatEther(txn?.amount)} Eth || Approval: ${
+                    txn?.approvals
+                  } || Sent: ${txn?.sent}`}</li>
+                ))}
+              </ul>
+            )}
```

Comment:
This is not styled yet . 

![Screenshot 2023-12-11 at 6 39 41 PM](https://github.com/warrenseah/multi-sig-wallet-frontend/assets/6447221/15d093a8-395a-4488-95c4-9a9b61efde56)

To add one item to the approval list , here are the steps 

Make sure you have loaded the deployer address 
0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266

Go to Only Owner Actions and fill in fields 
To Address: any address
Amount: 0.01 

Click create button and once txn is done , you should see a list item in the above. 

<img width="1340" alt="Screenshot 2023-12-11 at 6 54 41 PM" src="https://github.com/warrenseah/multi-sig-wallet-frontend/assets/6447221/5a5a6ced-d465-4087-af2b-f45d794a06e7">